### ISSUE TITLE
(Feature Request) - Accessibility Testing support

### ISSUE THREAD

**Issue by sfabriece** [angular/angular-cli#6572]
### Bug Report or Feature Request (mark with an `x`)
```
- [x ] feature request
```

### Desired functionality.
**Accessibility testing**

Is it possible to include tools like [axe](https://github.com/dequelabs/axe-cli) or [pa11y](https://github.com/pa11y/pa11y)
 for accessibility testing into the CLI?

**Comment by filipesilva** [comment]
Is there anything that stops you from adding them to a project right now? We try not to add extra tooling dependencies if possible, but if there's something that breaks usage of those tools maybe we can fix it.

**Comment by sfabriece** [comment]
No there is nothing stopping me from adding it currently. I just think that accessibility testing should be taken seriously and having it baked in into the CLI would reduce the friction to getting started, but I also understand that it might not a priority at the moment.

**Comment by filipesilva** [comment]
Maybe we could add something as a flag e.g `ng e2e --accessibility`.

**Comment by LukePettway** [comment]
I agree with @sfabriece , this is also something I would like to see as well.

**Comment by sfabriece** [comment]
@filipesilva that would be definitely welcome. Not to mix issues but maybe adding something like Lighthouse that supports customization would be useful as well. Then accessibility would be added as part of it. Just a thought, but not sure if the added functionality is needed.
